docs/concepts/data-partitions.md

```diff
@@ -0,0 +1,95 @@
+# Overview of Data Partitions
+
+Data partitioning is an optional feature that can be enabled for a DICOM service. It implements a light-weight data partition scheme that enables customers to store multiple copies of the same image with the same identifying instance UIDs on a single DICOM service. 
+
+While UIDs **should** be [unique across all contexts](http://dicom.nema.org/dicom/2013/output/chtml/part05/chapter_9.html), it's common practice for DICOM files to be written to portable storage media by a healthcare provider and given to a patient, who then gives the files to another healthcare provider, who then transfers the files into a new DICOM storage system. Thus, multiple copies of one DICOM file commonly exist in isolated DICOM systems. As DICOM functionality moves to the cloud, unifying previously disconnected systems, data partitioning can provide an on-ramp for your existing data stores and workflows.
+
+## Feature Enablement
+The data partitions feature can be enabled by setting the configuration key `DicomServer:Features:EnableDataPartitions` to `true` through your local [appsettings.json](../../src/Microsoft.Health.Dicom.Web/appsettings.json) file or host-specific options.
+
+Once enabled, this feature adds a new data partition resource, creates an initial partition named `Microsoft.Default`, and modifies the API surface of the DICOM server. 
+
+> *The data partition feature **cannot be disabled** once partitioned data is present - a `DataPartitionsFeatureCannotBeDisabledException` will be thrown at startup.*
+
+## API Changes
+All following URIs assume an implicit DICOM service base URI. For example, the base URI of a DICOM server running locally would be `https://localhost:63838/`. Example requests for new and modified APIs can be found in the [data partition Postman collection](../resources/data-partition-feature.postman_collection.json).
+
+### List Partitions
+Lists all data partitions.
+
+```http
+GET /partitions
+```
+
+#### Request Header
+
+| Name         | Required  | Type   | Description                     |
+| ------------ | --------- | ------ | ------------------------------- |
+| Content-Type | False     | string | `application/json` is supported |
+
+#### Responses
+
+| Name              | Type                          | Description                           |
+| ----------------- | ----------------------------- | ------------------------------------- |
+| 200 (OK)          | [Partition](#partition)`[]`   | A list of partitions is returned      |
+| 204 (No Content)  |                               | No partitions exist                   |
```

Comment:
Not necessarily anything we have to change, but I would have expected an empty array on no partitions existing.

Reply:
Good point, and something we should fix. It should never actually be returned since the schema upgrade creates the default `Microsoft.Default` partition.

Reply:
AB#86793